x-pack/plugins/index_lifecycle_management/public/extend_index_management/index.tsx

```diff
@@ -9,21 +9,24 @@ import { get, every, some } from 'lodash';
 import { i18n } from '@kbn/i18n';
 import { EuiSearchBar } from '@elastic/eui';
 
+import { ApplicationStart } from 'kibana/public';
 import { retryLifecycleForIndex } from '../application/services/api';
 import { IndexLifecycleSummary } from './components/index_lifecycle_summary';
 import { AddLifecyclePolicyConfirmModal } from './components/add_lifecycle_confirm_modal';
 import { RemoveLifecyclePolicyConfirmModal } from './components/remove_lifecycle_confirm_modal';
 
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { ExtensionsSetup } from '../../../index_management/public/services';
```

Comment:
I think if you export `ExtensionsSetup` from `x-pack/plugins/index_management/public/index.ts` you can remove the eslint-disable.